utils/unittest/execution_state.go

```diff
@@ -87,10 +87,10 @@ func genesisCommitHexByChainID(chainID flow.ChainID) string {
 		return GenesisStateCommitmentHex
 	}
 	if chainID == flow.Testnet {
-		return "6de2175363c8136ce966759fcc7a744d1b7f49d0d4dbf9a39970c82e29430b13"
+		return "29611cdf77990953d087ed0dc774293853a75c2c8da24bd4a268180521074ff0"
```

Comment:
why does this change end up changing the Testnet genesis state commit?

Reply:
Because of the changes in the `contract.cdc` file. I have updated the documentation. Any change in the EVM contract (or service contracts) results in a genesis state commit change for all networks.